corehq/apps/hqcase/templates/hqcase/explode_cases.html

```diff
@@ -64,35 +59,16 @@ <h2><small>Use at your own risk 🙃</small></h2>
         <div id="delete-exploded">
             <h1>Delete exploded cases</h1>
             <h2><small>Delete all cases created by previous explosions</small></h2>
+            <p>Find this ID on one of your exploded cases under the property 'cc_explosion_id'</p>
             <div class="row">
                 <div class="col-sm-8">
-                    <table class="table table-striped">
-                        <thead>
-                            <tr>
-                                <th>Explosion ID</th>
-                                <th>Case Count</th>
-                                <th>Delete</th>
-                            </tr>
-                        </thead>
-                        <tbody>
-                            {% for explosion_id, count in previous_explosions %}
-                                <tr>
-                                    <td>{{ explosion_id }}</td>
-                                    <td>{{ count }}</td>
-                                    <td>
-                                        <form class="form" action="" method="POST">
-                                            {% csrf_token %}
-                                            <input id="explosion_id" name="explosion_id" type="hidden" value="{{ explosion_id }}" />
-                                            <button type="submit" class="btn btn-danger">
-                                                <i class="fa fa-trash"></i>
-                                            </button>
-                                        </form>
-                                    </td>
-                                </tr>
-                            {% endfor %}
-                        </tbody>
-
-                    </table>
+                    <form class="form" action="" method="POST">
+                        {% csrf_token %}
+                        <input id="explosion_id" name="explosion_id" type="text" value="{{ explosion_id }}" placeholder="cc_explosion_id" />
```

Comment:
Where is `explosion_id` coming from? I don't see it in `get_context_data`.

Reply:
Sorry about that.